regtests/bin/matrix_ukmo_cray

```diff
@@ -104,9 +104,9 @@ fi
 
   if [ "$cmplOption" = 'y' ]
   then
-     export rtst="./bin/run_test -c $cmplr -S"
+     export rtst="./bin/run_test -o both -c $cmplr -S -N"
```

Comment:
@ukmo-ccbunney please indicate if you are okay with this change

Reply:
@JessicaMeixner-NOAA @mickaelaccensi - Yes, that's fine with me. Thanks for checking.

Reply:
Just to clarify, I am happy with the addition of the `-N` flag here and agnostic about the `-o both` flag (happy for it to be either here or in the matrix.base)